tests/bson/bson-generate-document-id.phpt

```diff
@@ -48,23 +48,23 @@ array(3) {
     string(8) "New York"
   }
   ["_id"]=>
-  object(%s\ObjectID)#%d (%d) {
+  object(%s\ObjectID)#%d (1) {
```

Comment:
no need. If anything -- these should be instead be replaced with %d everywhere :)